stacks_queues/stack.py

```diff
@@ -21,18 +23,23 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if not self.store:
+            raise StackEmptyException
+        return self.store.remove_last()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        if self.store.empty():
+            return True
+        else:
+            return False
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        return str(self.store)
```

Comment:
Nice reuse of the LinkedList string function. But notice that we need to store items at the front for it to print the contents in the desired order.